ice/agent.py

```diff
@@ -26,6 +26,7 @@ def __init__(self, *args, **kwargs):
     "instruct": lambda: OpenAIAgent(),
     "instruct-reasoning": lambda: OpenAIReasoningAgent(),
     "instruct-reasoning-crowd": lambda: OpenAIReasoningAgent(num_workers=8),
+    "davinci:ft-ought-experiments:synthesisv2-2022-10-17-23-02-55": lambda: OpenAIAgent(model="davinci:ft-ought-experiments:synthesisv2-2022-10-17-23-02-55"),
```

Comment:
We should have a better way of handling FT models

I'm a bit hesitant to merge this nonsense